lib/ex_prompt.ex

```diff
@@ -59,26 +72,32 @@ defmodule ExPrompt do
     - Yes, yes, YES, Y, y
     - No, no, NO, N, n
 
-  In case that the answer is none of the above, it will prompt again until we do.
+  In case that the answer is none of the above, it will prompt again until we do or the default value if informed.
```

Comment:
Let's write it like this:
```
  In case that the answer is none of the above, it will prompt again until we do or the default value if it's present.
```